src/component/editor/element/index.tsx

```diff
@@ -17,7 +18,30 @@ export function Element({ attributes, children, element }: ElementProps) {
         return <ul {...attributes}>{children}</ul>;
       case "list-item":
         return <li {...attributes}>{children}</li>;
+      case "a":
+        return <LinkComponent {...props} />;
     }
   }
   return <div {...attributes}>{children}</div>;
 }
+
+const LinkComponent = ({ attributes, children, element }: ElementProps) => {
+  const selected = useSelected();
+  return (
+    <a
+      {...attributes}
+      href={(element as LinkElement).url}
+      style={{
+        textDecoration: "none",
+        color: "blue",
+        border: selected ? "1px solid #00bcd4" : "none",
```

Comment:
I think there's a way to reference Plasmic style tokens in our code. Could you look into this?

Could you change the color to our teal color?

Also - could you remove the border style?

Reply:
hey @marcusellison, I just changed the color to ventureGreen